Makefile

```diff
@@ -61,38 +61,33 @@ ifeq (SIGN, 1)
 endif
 	@echo "ZREPL RELEASE ARTIFACTS AVAILABLE IN artifacts/release"
 
-srpm: $(ARTIFACTDIR)
+rpm: $(ARTIFACTDIR)
 ifneq (,$(filter $(HOST_OS_ID),fedora centos rhel))
-	$(info Generating srpm/rpm package requires the rpm build utilities to\
+	$(info Generating rpm package requires the rpm build utilities to\
 	       be installed. To install the minimal set of required packages:)
 	$(info )
 	$(info dnf install rpm-build rpmdevtools)
 	$(info )
 else
-	$(error Building srpm/rpm packages has only been tested on Fedora and \
+	$(error Building rpm packages has only been tested on Fedora and \
 	        is expected to work on RedHat/CentOS >= 8. Current host OS is:\
 	        $(HOST_OS_NAME))
 endif
-ifneq (,$(findstring -g,$(_ZREPL_VERSION)))
-	$(eval _ZREPL_RPM_GIT_COMMIT := HEAD)
-	$(eval _ZREPL_RPM_VERSION := g$(shell git rev-parse --short HEAD))
-else
-	$(eval _ZREPL_RPM_GIT_COMMIT := v$(_ZREPL_VERSION))
-	$(eval _ZREPL_RPM_VERSION := $(_ZREPL_VERSION))
-endif
-	rpmdev-setuptree
+	$(eval _ZREPL_RPM_VERSION := $(subst -,.,$(_ZREPL_VERSION)))
+	$(eval SRC_TARBALL_OUT := "$(HOME)/rpmbuild/SOURCES/zrepl-$(_ZREPL_RPM_VERSION).tar.gz")
```

Comment:
This directory is not guaranteed to be present as it is only created when running `rpmdev-setuptree`. Also, it voids your attempt to limit changes to the checkout directory. The original approach is altogether more elegant and easier to follow as everything related to a specific rpm build (be it for a release or development version) happens in a properly named sub-directory of ~/rpmbuild.

Reply:
`SRC_TARBALL_OUT` was an oversight from an earlier experiment

Reply:
> The original approach is altogether more elegant and easier to follow as everything related to a specific rpm build (be it for a release or development version) happens in a properly named sub-directory of ~/rpmbuild.

Agreed, but at the expense of having to re-build zrepl (and the docs, which we haven't addressed yet, twice).
See https://github.com/zrepl/zrepl/pull/369#issuecomment-683444372